### PR TITLE
[ZH] Update `Upgrading to lazer`

### PR DESCRIPTION
**Pull request by YumeMuzi**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Help_centre/Upgrading_to_lazer/zh.md [hunk 1]
```diff
@@ -42,22 +37,33 @@ osu!(lazer) 是游戏的下一个大更新。这凝结了几年以来精心幕
...
+| 软删除 | ![不支持][false] | ![支持][true][^soft-deletion] |
+| 游玩时立即更改并应用设置 | ![不支持][false] | ![支持][true]  |
+
+### 游玩
 
 | 功能 | stable | lazer |
 | :-- | :-- | :-- |
-| 难度调整 (DA) 模组 | ![No][false] | ![Yes][true][^difficulty-adjust] |
+| 调整谱面难度 | ![No][false] | ![Yes][true][^difficulty-adjust] |
 | 精准的表现分 (pp) 显示 | ![部分支持][partial][^online]  | ![支持][true]  |
```
Comment (CloneWith) [inline wiki/Help_centre/Upgrading_to_lazer/zh.md:57]: 56、57 两行的顺序似乎与原文不同，可调换一下

File: wiki/Help_centre/Upgrading_to_lazer/zh.md [hunk 2]
```diff
@@ -42,22 +37,33 @@ osu!(lazer) 是游戏的下一个大更新。这凝结了几年以来精心幕
...
 | 精细调整每个模组 | ![不支持][false]  | ![支持][true]  |
 | 新的“娱乐”模组 | ![不支持][false]  | ![支持][true]  |
 | 标准化连击色[^normalisation] | ![不支持][false]  | ![支持][true]  |
 | 按住显示 HUD | ![不支持][false]  | ![支持][true][^hold-for-hud] |
-| 偏移值 (offset) 校准 | ![部分支持][partial][^offset-calibration-stable] | ![支持][true][^offset-calibration-lazer] |
+| 按谱面校准偏移值 (offset) | ![部分支持][partial][^offset-calibration-stable] | ![支持][true][^offset-calibration-lazer] |
 | 光标拖动 osu! 滑条时，滑条“蛇形”缩回 | ![不支持][false]  | ![支持][true][^can-disable]  |
 | 对 osu! 玩家稍微友好的“物件锁 (note lock)” | ![不支持][false]  | ![支持][true][^note-lock]  |
-| osu!mania 基于时间轴给物件上色 | ![不支持][false]  | ![支持][true]  |
+| osu!mania、osu! 基于时间轴给物件上色 | ![不支持][false]  | ![支持][true]  |
```
Comment (CloneWith) [inline wiki/Help_centre/Upgrading_to_lazer/zh.md:66]: 下面缺失了两行 回放控制和弹幕

File: wiki/Help_centre/Upgrading_to_lazer/zh.md
Comment (CloneWith) [inline wiki/Help_centre/Upgrading_to_lazer/zh.md:1]: 表格里一些文字末尾带两个空格的，感觉有些奇怪，是规范问题吗？

**Review by CloneWith — CHANGES_REQUESTED** [review]

File: wiki/Help_centre/Upgrading_to_lazer/zh.md
Comment (YumeMuzi) [reply wiki/Help_centre/Upgrading_to_lazer/zh.md:1]: 两个空格是之前翻译的问题吧，，按照规范，`|` 前后只能有1个空格

**Review by CloneWith — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**